src/systems/filecoin_markets/deal/_index.md

```diff
@@ -3,6 +3,52 @@ menuTitle: Deals
 title: Market Deals
 ---
 
-There are two types of deals in Filecoin markets, storage deals and retrieval deals. Storage deals are recorded on the blockchain and enforced by the protocol. Retrieval deals are off chain and enabled by micropayment channel by transacting parties. All deal negotiation happen off chain and a request-response style storage deal protocol is in place to submit agreed-upon storage deals onto the network with `CommitSector` to gain storage power on chain. Hence, there is a `StorageDealProposal` and a `RetrievalDealProposal` that are half sign contracts submitted by clients to be counter-signed and posted on-chain by the miners. 
+There are two types of deals in Filecoin markets, storage deals and retrieval deals. Storage deals are recorded on the blockchain and enforced by the protocol. Retrieval deals are off chain and enabled by micropayment channel by transacting parties. All deal negotiation happen off chain and a request-response style storage deal protocol is in place to submit agreed-upon storage deals onto the network with `CommitSector` to gain storage power on chain. Hence, there is a `StorageDealProposal` and a `RetrievalDealProposal` that are half-signed contracts submitted by clients to be counter-signed and posted on-chain by the miners.
+
+Filecoin Storage Market Deal Flow
+
+# Add Storage Deal and Power
+
+- 1. `StorageClient` and `StorageProvider` call `StorageMarketActor.AddBalance` to deposit funds into Storage Market. There are two fund states in the Storage Market, `Locked` and `Available`.
+    - `StorageClient` and `StorageProvider` can call `WithdrawBalance` before any deal is made. (move to state X)
```

Comment:
The should be able to call `WithdrawBalance` at any time, regardless of deals, and withdraw any unlocked funds.